apps/oxlint/fixtures/print_config/ban_rules/expect.json

```diff
@@ -23,6 +23,7 @@
       "rootDir": []
     },
     "react": {
+      "pragma": "React",
```

Comment:
Why not keep same with ESLint config file? https://github.com/oxc-project/oxc/blob/3923e63b6100104a9985dec6fa27780471e88d78/crates/oxc_linter/src/config/oxlintrc.rs#L16

Reply:
`form_components` and `link_components` don't seem to be in `react` but we designed them this way from the beginning.
```rust
// <https://github.com/jsx-eslint/eslint-plugin-react#configuration-legacy-eslintrc->
#[derive(Debug, Clone, Deserialize, Serialize, JsonSchema)]
#[cfg_attr(test, derive(PartialEq))]
pub struct ReactPluginSettings {
    #[serde(default = "default_pragma")]
    pub pragma: CompactStr,

    #[serde(default)]
    #[serde(rename = "formComponents")]
    form_components: Vec<CustomComponent>,

    #[serde(default)]
    #[serde(rename = "linkComponents")]
    link_components: Vec<CustomComponent>,
    // TODO: More properties should be added
}
```

Reply:
https://github.com/jsx-eslint/eslint-plugin-react#configuration-legacy-eslintrc

Reply:
My fault, I missed this. Seems a simple way to fix this issue has merged.